languages/rust/exercises/concept/string-vs-str/.meta/example.rs

```diff
@@ -0,0 +1,35 @@
+pub struct CSVBuilder {
+    content: String,
+}
+
+impl CSVBuilder {
+    // Create a new builder
+    pub fn new() -> Self {
+        Self { content: String::new() }
+    }
+
+    /// Adds an item to the list separated by a space and a comma.
+    pub fn add(&mut self, val: &str) {
+        if !self.content.is_empty() {
+            self.content.push(',');
+        }
+
+        if val.contains(",") || val.contains("\"") || val.contains("\n") {
+            self.content.push('"');
+            self.content.push_str(&val.replace("\"", "\"\""));
```

Comment:
```suggestion
        if val.contains(",") || val.contains(r#"""#) || val.contains("\n") {
            self.content.push('"');
            self.content.push_str(&val.replace(r#"""#, r#""""#));
```

Nit: using raw literals reads a little more cleanly to me.